integrations/extensions/starter-kits/metaweather/readme.md

```diff
@@ -43,9 +43,39 @@ To get started with using the OpenAPI spec, you can upload the `metaweather.adva
       - woeid: 1. OK, I can help with that. What is your ...
       - year: year
       - month: month
+      - date: date
     ```
 If you want to know the Where-On-Earth-ID of a particular location, simply use the first action to query using the location name. The Where-On-Earth-ID of the location will be returned to you.
 
+### Advanced: Blog Spec
```

Comment:
```suggestion
### Even More Advanced: How to use Metaweather in conjunction with another extension
```

Reply:
Edited to say: "Even More Advanced: Using the MetaWeather Extension with Other Extensions"